deploy/auto-install.sh

```
@@ -172,14 +169,14 @@ setup_docker_openwisp() {
 	else
-		mv $ENV_USER $INSTALL_PATH/.env &>>$LOG_FILE
-		rm -rf $ENV_USER &>>$LOG_FILE
+		mv "$ENV_USER" "$INSTALL_PATH/.env" &>>"$LOG_FILE"
+		rm -rf "$ENV_USER" &>>"$LOG_FILE"
 	fi
 
 	start_step "Configuring docker-openwisp..."
 	report_ok
 	start_step "Starting images docker-openwisp (this will take a while)..."
-	make start TAG=$(cat $INSTALL_PATH/VERSION) -C $INSTALL_PATH/ &>>$LOG_FILE
+	make start TAG=$(cat "$INSTALL_PATH/VERSION") -C "$INSTALL_PATH/" &>>"$LOG_FILE"
```

Shellcheck (0.11.0)

[warning] Quote this to prevent word splitting.

(SC2046)